test/ProjectService.Data.UnitTests/GetUserProjectsRepositoryTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Provider;
+using LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.ProjectService.Data.UnitTests
+{
+    public class GetUserProjectsRepositoryTests
+    {
+        private IDataProvider provider;
+        private IProjectRepository repository;
+
+        private Guid userIdWithOneProject;
+        private Guid userIdWithTwoProjects;
+        private Guid userIdWithoutActiveProjects;
+
+        private DbProject dbProject1;
+        private DbProject dbProject2;
+        private DbProject dbNotActiveProject;
+
+        private DbProjectUser dbUserWithOneProject;
+        private DbProjectUser dbUserWithTwoProjects1;
+        private DbProjectUser dbUserWithTwoProjects2;
+        private DbProjectUser dbUserWithoutActiveProject;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var dbOptions = new DbContextOptionsBuilder<ProjectServiceDbContext>()
+                .UseInMemoryDatabase("InMemoryDatabase")
+                .Options;
+
+            provider = new ProjectServiceDbContext(dbOptions);
+            repository = new ProjectRepository(provider);
+
+            userIdWithOneProject = Guid.NewGuid();
+            userIdWithTwoProjects = Guid.NewGuid();
+            userIdWithoutActiveProjects = Guid.NewGuid();
+
+            dbUserWithOneProject = new DbProjectUser
+            {
+                Id = Guid.NewGuid(),
+                UserId = userIdWithOneProject,
+                IsActive = true
+            };
+
+            dbUserWithTwoProjects1 = new DbProjectUser
+            {
+                Id = Guid.NewGuid(),
+                UserId = userIdWithTwoProjects,
+                IsActive = true
+            };
+
+            dbUserWithTwoProjects2 = new DbProjectUser
+            {
+                Id = Guid.NewGuid(),
+                UserId = userIdWithTwoProjects,
+                IsActive = true
+            };
+
+            dbUserWithoutActiveProject = new DbProjectUser
+            {
+                Id = Guid.NewGuid(),
+                UserId = userIdWithoutActiveProjects,
+                IsActive = true
+            };
+
+            dbProject1 = new DbProject
+            {
+                Id = Guid.NewGuid(),
+                IsActive = true,
+                Name = "Prroject1"
+            };
```

Comment:
use OneTimeSetUp

Reply:
In this case, I am breaking several tests in this direction.